app/api/graphql/route.ts

```diff
@@ -98,11 +98,19 @@ const resolvers: { Query: QueryResolvers; Mutation: MutationResolvers } = {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     memos: async (_parent, _args, _context) => {
       try {
-        // Use public client for queries - no authentication needed
-        // Default to 'metrue' owner, but could be made configurable
-        const client = createPublicGitHubClient('metrue')
-        const result = await client.getMemos()
-        return Array.isArray(result) ? result : []
+        // In development, use local client; in production, use GitHub client
+        if (process.env.NODE_ENV === 'development') {
```

Comment:
Please ensure that when we build and run in the production, the env.NODE_ENV is not 'development', it's production?